integrasjon/kafka-properties/src/main/java/no/nav/vedtak/felles/integrasjon/kafka/KafkaProperties.java

```diff
@@ -82,46 +78,31 @@ public static String getAvroSchemaRegistryBasicAuth() {
     }
 
-
-    private static String getAivenConfig(AivenProperty property) {
-        return Optional.ofNullable(ENV.getProperty(property.name()))
-            .orElseGet(() -> ENV.getProperty(property.name().toLowerCase().replace('_', '.')));
-    }
-
     private static String generateClientId() {
         return APPLICATION_NAME + "-" + UUID.randomUUID();
     }
 
     private static void putSecurity(Properties props) {
-        if (IS_DEPLOYMENT_OR_KAFKA_DOCKER) {
-            props.put(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, SecurityProtocol.SSL.name);
-            props.put(SslConfigs.SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG, "");
-            props.put(SslConfigs.SSL_TRUSTSTORE_TYPE_CONFIG, "jks");
-            props.put(SslConfigs.SSL_TRUSTSTORE_LOCATION_CONFIG, getAivenConfig(AivenProperty.KAFKA_TRUSTSTORE_PATH));
-            props.put(SslConfigs.SSL_TRUSTSTORE_PASSWORD_CONFIG, hentPassordTruststore());
-            props.put(SslConfigs.SSL_KEYSTORE_TYPE_CONFIG, "PKCS12");
-            props.put(SslConfigs.SSL_KEYSTORE_LOCATION_CONFIG, getAivenConfig(AivenProperty.KAFKA_KEYSTORE_PATH));
-            props.put(SslConfigs.SSL_KEYSTORE_PASSWORD_CONFIG, getAivenConfig(AivenProperty.KAFKA_CREDSTORE_PASSWORD));
-        } else { // Deprecated: Fjern etter nytt kafka image
-            props.setProperty(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, SecurityProtocol.SASL_SSL.name);
-            props.setProperty(SaslConfigs.SASL_MECHANISM, "PLAIN");
-            String jaasTemplate = "org.apache.kafka.common.security.scram.ScramLoginModule required username=\"%s\" password=\"%s\";";
-            String jaasCfg = String.format(jaasTemplate, "vtp", "vtp");
-            props.setProperty(SaslConfigs.SASL_JAAS_CONFIG, jaasCfg);
-        }
+        props.put(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, SecurityProtocol.SSL.name);
+        props.put(SslConfigs.SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG, "");
+        props.put(SslConfigs.SSL_TRUSTSTORE_TYPE_CONFIG, "jks");
+        props.put(SslConfigs.SSL_TRUSTSTORE_LOCATION_CONFIG, getAivenConfig(AivenProperty.KAFKA_TRUSTSTORE_PATH));
+        props.put(SslConfigs.SSL_TRUSTSTORE_PASSWORD_CONFIG, hentPassordTruststore());
+        props.put(SslConfigs.SSL_KEYSTORE_TYPE_CONFIG, "PKCS12");
+        props.put(SslConfigs.SSL_KEYSTORE_LOCATION_CONFIG, getAivenConfig(AivenProperty.KAFKA_KEYSTORE_PATH));
+        props.put(SslConfigs.SSL_KEYSTORE_PASSWORD_CONFIG, getAivenConfig(AivenProperty.KAFKA_CREDSTORE_PASSWORD));
+    }
+
+    private static String getAivenConfig(AivenProperty property) {
+        return Optional.ofNullable(ENV.getProperty(property.name()))
+            .orElseGet(() -> ENV.getProperty(property.name().toLowerCase().replace('_', '.')));
     }
 
     private static String hentPassordTruststore() {
-        if (brukKafkaAivenPropertyLokalt()) {
+        if (ENV.isLocal()) {
             return ENV.getTruststorePassword();
```

Comment:
Det som hadde vært best er om vi kunne hatt samme passord for truststore og keystore, aka KAFKA_CREDSTORE_PASSWORD blir brukt begge plasser, slik at det blir helt likt prod-kode.

Dere kan jo se om det er mulig å sette det slik at samme passord brukes.